rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -58,3 +62,79 @@ func (pal *enclaveRuntimePal) Destroy() error {
 	api := &enclaveRuntimePalApiV1{}
 	return api.destroy()
 }
+
+func (pal *enclaveRuntimePal) GetSgxReport(targetInfo []byte, data []byte) ([]byte, error) {
+	api := &enclaveRuntimePalApiV3{}
+	return api.GetSgxReport(targetInfo, data)
+}
+
+func parseAttestParameters(Spid string, Product string, Subscription_key string) map[string]string {
```

Comment:
Rename the variable Subscription_key to subescriptionKey. Also, other parameters should not be capital letter.